cmd/dep/testdata/harness_tests/init/govendor/case1/final/Gopkg.toml

```diff
@@ -0,0 +1,9 @@
+ignored = ["github.com/sdboyer/dep-test"]
+
+[[constraint]]
+  name = "github.com/carolynvs/go-dep-test"
```

Comment:
This should have been ignored because it is only referenced by an ignored package `samples`, and should not have ended up in the manifest, lock or vendor.